src/test/resources/jdl/reflexive_relationship-liquibase-changeset.yaml

```diff
@@ -21,7 +21,7 @@ databaseChangeLog:
                               constraints:
                                   nullable: false
                               name: amount
-                              type: decimal
+                              type: DECIMAL(19, 5)
```

Comment:
What is the reason behind this?
Does oracle not provide default precision/scale?

Reply:
Yes, if I didn't change it would just be `NUMBER` without length. I needed a `NUMBER(19,5)` field type for Big Decimal.

Reply:
ok but it is possible to have any kind of precision/scale -> NUMBER(21, 9), right? Which seems related to hard-coded `parseSqlType(..)`